pkg/s3utils/utils.go

```diff
@@ -122,7 +122,7 @@ func IsAmazonChinaEndpoint(endpointURL url.URL) bool {
 	if endpointURL == sentinelURL {
 		return false
 	}
-	return endpointURL.Host == "s3.cn-north-1.amazonaws.com.cn"
+	return strings.HasSuffix(endpointURL.Host, ".amazonaws.com.cn")
```

Comment:
The comment only talks about `Beijing` region. How about the newly added `China (Ningxia)` region?
Don't we need to explain covered regions and hence this function returns `true` either the region is `Beijing` or `Ningxia` ?